Fails to build on main without bevy_mesh/morph feature

## Bevy version and features

af5f307dd

> cargo build --no-default-features --features std,x11,bevy_winit,bevy_state,bevy_window,bevy_pbr,bevy_sprite,bevy_text,bevy_core_pipeline,bevy_picking,bevy_animation,png,zstd_rust,tonemapping_luts,ktx2


```
bevy on  main [$?] is 📦 v0.18.0-dev via 🦀 v1.91.0-nightly 
❯ cargo build --no-default-features --features std,x11,bevy_winit,bevy_state,bevy_window,bevy_pbr,bevy_sprite,bevy_text,bevy_core_pipeline,bevy_picking,bevy_animation,png,zstd_rust,tonemapping_luts,ktx2
   Compiling bevy_pbr v0.18.0-dev (D:\Repos\Games\bevy\crates\bevy_pbr)
error[E0432]: unresolved import `bevy_mesh::morph`
  --> crates\bevy_pbr\src\render\mesh_bindings.rs:4:16
   |
 4 | use bevy_mesh::morph::MAX_MORPH_WEIGHTS;
   |                ^^^^^ could not find `morph` in `bevy_mesh`
   |
note: found an item that was configured out
  --> D:\Repos\Games\bevy\crates\bevy_mesh\src\lib.rs:13:9
   |
12 | #[cfg(feature = "morph")]
   |       ----------------- the item is gated behind the `morph` feature
   |                ^^^^^ could not find `morph` in `bevy_mesh`
   |
note: found an item that was configured out
  --> D:\Repos\Games\bevy\crates\bevy_mesh\src\lib.rs:13:9
   |
12 | #[cfg(feature = "morph")]
   |       ----------------- the item is gated behind the `morph` feature
13 | pub mod morph;
   |         ^^^^^

error[E0432]: unresolved import `bevy_mesh::morph`
  --> crates\bevy_pbr\src\render\morph.rs:5:16
   |
 5 | use bevy_mesh::morph::{MeshMorphWeights, MAX_MORPH_WEIGHTS};
   |                ^^^^^ could not find `morph` in `bevy_mesh`
   |
note: found an item that was configured out
  --> D:\Repos\Games\bevy\crates\bevy_mesh\src\lib.rs:13:9
   |
12 | #[cfg(feature = "morph")]
   |       ----------------- the item is gated behind the `morph` feature
13 | pub mod morph;
   |         ^^^^^

error[E0599]: no method named `with_generated_tangents` found for struct `bevy_mesh::Mesh` in the current scope
  --> crates\bevy_pbr\src\decal\forward.rs:37:18
   |                ^^^^^ could not find `morph` in `bevy_mesh`
   |
note: found an item that was configured out
  --> D:\Repos\Games\bevy\crates\bevy_mesh\src\lib.rs:13:9
   |
12 | #[cfg(feature = "morph")]
   |       ----------------- the item is gated behind the `morph` feature
13 | pub mod morph;
   |         ^^^^^

error[E0432]: unresolved import `bevy_mesh::morph`
  --> crates\bevy_pbr\src\render\morph.rs:5:16
   |
 5 | use bevy_mesh::morph::{MeshMorphWeights, MAX_MORPH_WEIGHTS};
   |                ^^^^^ could not find `morph` in `bevy_mesh`
   |
   |         ^^^^^

error[E0432]: unresolved import `bevy_mesh::morph`
  --> crates\bevy_pbr\src\render\morph.rs:5:16
   |
 5 | use bevy_mesh::morph::{MeshMorphWeights, MAX_MORPH_WEIGHTS};
   |                ^^^^^ could not find `morph` in `bevy_mesh`
   |
note: found an item that was configured out
  --> D:\Repos\Games\bevy\crates\bevy_mesh\src\lib.rs:13:9
   |
12 | #[cfg(feature = "morph")]
   |       ----------------- the item is gated behind the `morph` feature
13 | pub mod morph;
   |         ^^^^^

error[E0599]: no method named `with_generated_tangents` found for struct `bevy_mesh::Mesh` in the current scope
  --> crates\bevy_pbr\src\decal\forward.rs:37:18
   |
33 | /             Rectangle::from_size(Vec2::ONE)
34 | |                 .mesh()
35 | |                 .build()
36 | |                 .rotated_by(Quat::from_rotation_arc(Vec3::Z, Vec3::Y))
37 | |                 .with_generated_tangents()
   | |                 -^^^^^^^^^^^^^^^^^^^^^^^ method not found in `bevy_mesh::Mesh`
   | |_________________|
   |

error[E0609]: no field `morph_targets` on type `&RenderMesh`
    --> crates\bevy_pbr\src\render\mesh.rs:2822:45
   |         ^^^^^

error[E0432]: unresolved import `bevy_mesh::morph`
  --> crates\bevy_pbr\src\render\morph.rs:5:16
   |
 5 | use bevy_mesh::morph::{MeshMorphWeights, MAX_MORPH_WEIGHTS};
   |                ^^^^^ could not find `morph` in `bevy_mesh`
   |
note: found an item that was configured out
  --> D:\Repos\Games\bevy\crates\bevy_mesh\src\lib.rs:13:9
   |
12 | #[cfg(feature = "morph")]
   |       ----------------- the item is gated behind the `morph` feature
13 | pub mod morph;
   |         ^^^^^

error[E0599]: no method named `with_generated_tangents` found for struct `bevy_mesh::Mesh` in the current scope
  --> crates\bevy_pbr\src\decal\forward.rs:37:18
   |
33 | /             Rectangle::from_size(Vec2::ONE)
34 | |                 .mesh()
35 | |                 .build()
36 | |                 .rotated_by(Quat::from_rotation_arc(Vec3::Z, Vec3::Y))
   |                ^^^^^ could not find `morph` in `bevy_mesh`
   |
note: found an item that was configured out
  --> D:\Repos\Games\bevy\crates\bevy_mesh\src\lib.rs:13:9
   |
12 | #[cfg(feature = "morph")]
   |       ----------------- the item is gated behind the `morph` feature
13 | pub mod morph;
   |         ^^^^^

error[E0599]: no method named `with_generated_tangents` found for struct `bevy_mesh::Mesh` in the current scope
  --> crates\bevy_pbr\src\decal\forward.rs:37:18
   |
33 | /             Rectangle::from_size(Vec2::ONE)
34 | |                 .mesh()
35 | |                 .build()
36 | |                 .rotated_by(Quat::from_rotation_arc(Vec3::Z, Vec3::Y))
37 | |                 .with_generated_tangents()
   | |                 -^^^^^^^^^^^^^^^^^^^^^^^ method not found in `bevy_mesh::Mesh`
   | |_________________|
   |

error[E0609]: no field `morph_targets` on type `&RenderMesh`
    --> crates\bevy_pbr\src\render\mesh.rs:2822:45
     |
2822 |             if let Some(targets) = gpu_mesh.morph_targets.as_ref() {
     |                                             ^^^^^^^^^^^^^ unknown field
     |
     = note: available fields are: `vertex_count`, `buffer_info`, `key_bits`, `layout`

Some errors have detailed explanations: E0432, E0599, E0609.
For more information about an error, try `rustc --explain E0432`.
error: could not compile `bevy_pbr` (lib) due to 4 previous errors
```

This feature set worked on `v0.17.0-rc.2` I think

I believe the problem was introduced in 

https://github.com/bevyengine/bevy/pull/21389

--- a/crates/bevy_pbr/Cargo.toml
+++ b/crates/bevy_pbr/Cargo.toml
@@ -43,11 +43,16 @@ bevy_diagnostic = { path = "../bevy_diagnostic", version = "0.18.0-dev" }
 bevy_ecs = { path = "../bevy_ecs", version = "0.18.0-dev" }
 bevy_light = { path = "../bevy_light", version = "0.18.0-dev" }
 bevy_image = { path = "../bevy_image", version = "0.18.0-dev" }
-bevy_mesh = { path = "../bevy_mesh", version = "0.18.0-dev", features=["morph", "bevy_mikktspace"] }
+bevy_mesh = { path = "../bevy_mesh", version = "0.18.0-dev", features = [
+  "morph",
+  "bevy_mikktspace",
+] }
 bevy_shader = { path = "../bevy_shader", version = "0.18.0-dev" }
 bevy_math = { path = "../bevy_math", version = "0.18.0-dev" }
 bevy_reflect = { path = "../bevy_reflect", version = "0.18.0-dev" }
-bevy_render = { path = "../bevy_render", version = "0.18.0-dev", features=["morph"] }
+bevy_render = { path = "../bevy_render", version = "0.18.0-dev", features = [
+  "morph",
+] }
 bevy_camera = { path = "../bevy_camera", version = "0.18.0-dev" }
 bevy_tasks = { path = "../bevy_tasks", version = "0.18.0-dev", optional = true }
 bevy_transform = { path = "../bevy_transform", version = "0.18.0-dev" }

